PullRequestCreated/ReadMe.md

```diff
@@ -37,7 +37,7 @@ Jira project key that is used to create new issues.
 Additional fields to set when creating the Jira issue. 
 
 ```yaml
-additional-fields: '{ "priority": { "name": "Major" }'
+additional-fields: '{ "components": [ { "name": "Something" } ], "labels": ["Something"] }'
```

Comment:
A closing `}` was actually missing in the old code. I used `components` and `labels` here, as those were actually used in production and it will be handy to have an easy copy/paste source here